intelmq/bots/outputs/mongodb/output.py

```diff
@@ -43,7 +44,10 @@ def process(self):
         event = self.receive_message()
 
         try:
-            self.collection.insert(event.to_dict(hierarchical=self.parameters.hierarchical_output))
+            tmp_dict = event.to_dict(hierarchical=self.parameters.hierarchical_output)
+            tmp_dict["time"]["observation"] = dateutil.parser.parse(tmp_dict["time"]["observation"])
```

Comment:
sorry, no time this week. I'll do it next week

Reply:
unfortunately, using intelmq.lib.harmonization.DateTime.parse_utc_isoformat is not enough. The format returned is still a str. So pymongo insert a string instead of Isodate.
I don't see any way to avoid using dateutil

Reply:
Oh, ok. That method in the libs looks strange, I need to look at it later when I refactor that file.